modules/admin_manual/pages/configuration/files/encryption/master-key-encryption.adoc

```diff
@@ -2,21 +2,20 @@
 = Master Key Based Encryption
 Matthew Setter <matthew@matthewsetter.com>
 :toc: right
+:toclevels: 1
 :experimental:
 :keywords: master key based encryption, encryption
 :description: This guide will show you how to work with Master Key encryption in ownCloud. 
 
 == Introduction
 
-With Master Key based encryption, there is only one key (or key pair) and all files are encrypted using that key pair. 
-This is highly recommended for new instances to avoid restrictions in functionality of user key encryption.
+With Master Key based encryption, there is only one key (or key pair) and all files are encrypted
```

Comment:
Is there a specific need to split sentences over multiple lines?

Reply:
yes, unreadbility in editor,
you have to move editor slider to read the whole text
awful

Reply:
Which editor are you using?

Reply:
Can you post a screenshot?

Reply:
![image](https://user-images.githubusercontent.com/3321281/57317157-c5d7d480-70f7-11e9-9d96-2ba2b0193c9b.png)

Reply:
![screenshot](https://user-images.githubusercontent.com/196801/57361815-953a7e00-717d-11e9-9689-b9bbd09fd434.png)